var/spack/repos/builtin/packages/roms/package.py

```diff
@@ -71,7 +71,8 @@ def _edit_arch(self, spec, prefix, lib):
         """
         Edit Linux-flang.mk makefile to support AOCC compiler
         """
-        fflags = ['-fveclib=AMDLIBM', '-O3', '-ffast-math']
+        fflags = ['-fveclib=AMDLIBM', '-O3', '-ffast-math',
+                  '-funroll-loops', '-Mstack_arrays', '-std=f2008']
```

Comment:
@amd-toolchain-support Quick questions.  Why is `-Mstack_arrays` used instead of `-fstack-arrays`?  Is `stack_arrays` a directory for module files?

Reply:
(My Fortran is a bit rusty. :grin:)

Reply:
Hi @tldahlgren , `fstack-array` is a GNU flag and it is compiler specific.

Reply:
Hello @tldahlgren, can you please do the needful for this request.